### ISSUE TITLE
portable databroker does not refresh in other process

### ISSUE THREAD

**Issue by CJ-Wright** [bluesky/databroker#396]
It seems that if you insert data into a portable databroker in one process (input) and try to get it out in a different process (output) the output process does not refresh to reflect the new data unless the a new databroke object is created from the same config.

Example to follow, once I sort out the round trip of `db.from_config(db.get_config())` (see #395 )

**Comment by danielballan** [comment]
Do you have the bandwidth to dig a little deeper on this? Our resources are aimed the 1.0 rewrite.

**Comment by CJ-Wright** [comment]
Yes but I think the other PR needs to go in first (that way it is easier to copy databrokers across processes)
